### PR TITLE
Updated Repository Overview to mark meshery-smi-conformance-action as archived

### LINKED ISSUE #5832: [Handbook] Update Repository Overview to mark meshery-smi-conformance-action as archived
### Current Behavior
This repo is now archived - https://github.com/layer5io/meshery-smi-conformance-action

### Desired Situation
The https://layer5.io/community/handbook/repository-overview page should reflect this archived status.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)

- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by H-A-R-S-H-K**
**Description**

This PR fixes #5832 

Screenshots

Screenshot 2024-08-21 at 1 20 29 PM

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit acacf32cb0e24cc8c00e5ff3071855b954b1324c at: https://66c59d9068e154fb6a036ed4--layer5.netlify.app

File: src/sections/Community/Handbook/repository.js [hunk 1]
```diff
@@ -208,7 +208,7 @@ const Repository = () => {
                       </tr>
                     </thead>
                     {backendProject.subdata.map((subdata) => {
-                      const { project,image,language,description,repository,link,maintainers_name } = subdata;
+                      const { project,image,language,description,repository,link,maintainers_name,accessRequired } = subdata;
```
Comment (Ashparshp) [inline src/sections/Community/Handbook/repository.js:211]: @H-A-R-S-H-K, why these changes?

File: src/sections/Community/Handbook/repository.js [hunk 2]
```diff
@@ -236,6 +236,7 @@ const Repository = () => {
                               >
                                 <img className="github-icon" src={github} alt="github-icon" />
                               </a>
+                              <div className="accessRequired"> { accessRequired != "" ? accessRequired : ""}</div>
```
Comment (Ashparshp) [inline src/sections/Community/Handbook/repository.js:239]: ??

File: src/sections/Community/Handbook/repository.js (hunk 1 above)
Comment (H-A-R-S-H-K) [reply src/sections/Community/Handbook/repository.js:211]: @Ashparshp In the frontend projects subdata this type of code was followed so i thought to make it same for backend.

**Comment by Ashparshp** [comment]
@H-A-R-S-H-K Thanks for your contribution, let's discuss this on the website's call. Please add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

File: src/sections/Community/Handbook/repository.js (hunk 1 above)
Comment (sudhanshutech) [reply src/sections/Community/Handbook/repository.js:211]: is this was the ask @H-A-R-S-H-K ?

File: src/sections/Community/Handbook/repository.js (hunk 2 above)
Comment (sudhanshutech) [reply src/sections/Community/Handbook/repository.js:239]: @H-A-R-S-H-K explain this?

File: src/sections/Community/Handbook/repository.js (hunk 1 above)
Comment (H-A-R-S-H-K) [reply src/sections/Community/Handbook/repository.js:211]: @sudhanshutech well we have to show the status of SMI Conformance repo as archived. To achieve that I added a key "accessRequired" with value as "*archived" in the backend projects subdata and added a div to display that "*archived".

Screenshot 2024-08-24 at 2 33 43 PM

was this not the ask ?

File: src/sections/Community/Handbook/repository.js (hunk 2 above)
Comment (H-A-R-S-H-K) [reply src/sections/Community/Handbook/repository.js:239]: @sudhanshutech the reason why I made such changes is because the frontend projects subdata follows this type of code to display status of frontend projects such as "*program participation required" on https://layer5.io/community/handbook/repository-overview page.

**Comment by Ashparshp** [comment]
@H-A-R-S-H-K, You were unavailble in last meeting?

**Review by Ashparshp — COMMENTED** [review]
Please change the ask only, Also, please attend the meeting to get feedback on the timing.

Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT). Please consider adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk) || [meeting link](https://meet.layer5.io/websites).

**Comment by vishalvivekm** [comment]
@H-A-R-S-H-K 
Thank you, as `status` is not something we would want to represent as `accessRequired` we can introduce a new prop called `status` in repo-data.js and follow existing css and logic of `accessRequired`.

Alternatively, we could rename `accessRequired` to something more generic, like `additionalInfo`.
@H-A-R-S-H-K @Ashparshp will either of you follow up here?
